CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 - @cosmjs/cosmwasm: Rename `CosmWasmClient.postTx` method to `.broadcastTx`.
 - @cosmjs/cosmwasm: Rename `SigningCosmWasmClient.signAndPost` method to
   `.signAndBroadcast`.
+- @cosmjs/encoding: Add `limit` parameter to `Bech32.decode`. If unspecified,
+  this will now allow decoding of addresses which exceed the specification limit
+  of 90 characters.
```

Comment:
The describtion is hard to understand I thing. Suggestion: "The new default limit is infinity (was 90 before). Set it to 90 to create a strict decoder."